[libinput,libwacom] Unable to install

```
/bin/grep: /usr/local/lib/libudev.la: No such file or directory
/bin/sed: can't read /usr/local/lib/libudev.la: No such file or directory
libtool:   error: '/usr/local/lib/libudev.la' is not a valid libtool archive
make[2]: *** [Makefile:422: libwacom.la] Error 1
make[2]: Leaving directory '/media/tmp/crew/libwacom-0.28.tar.bz2.dir/libwacom-0.28/libwacom'
make[1]: *** [Makefile:461: all-recursive] Error 1
make[1]: Leaving directory '/media/tmp/crew/libwacom-0.28.tar.bz2.dir/libwacom-0.28'
make: *** [Makefile:370: all] Error 2
libwacom failed to build: `make V=0 -j1` exited with 2
```
Missing libudev dependency?

--- a/packages/libwacom.rb
+++ b/packages/libwacom.rb
@@ -3,11 +3,12 @@ require 'package'
 class Libwacom < Package
   description 'libwacom is a wrapper library for evdev devices.'
   homepage 'http://linuxwacom.sourceforge.net/wiki/index.php/Libwacom'
-  version '0.28'
+  version '0.28-0'
   source_url 'https://downloads.sourceforge.net/linuxwacom/libwacom-0.28.tar.bz2'
   source_sha256 'e7d632301288b221cb5af69b4c5e57fd062bafd9a9acd6f9ce271570103267ef'
 
   depends_on 'libgudev'
+  depends_on 'eudev'
        
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

